### ISSUE TITLE
install script has unknown flag and failed to install 1.12.1 version

### ISSUE THREAD

**Issue by LiamLea** [kubeovn/kube-ovn#3331]
# Actual Behavior
* kube-ovn-cni pods are in CrashLoopBackOff status
```shell
kube-system   kube-ovn-cni-8lx2p                     0/1     CrashLoopBackOff   6 (2m16s ago)     7m58s
kube-system   kube-ovn-cni-d4xzn                     0/1     CrashLoopBackOff   6 (118s ago)      7m58s
kube-system   kube-ovn-cni-qmfpk                     0/1     CrashLoopBackOff   6 (119s ago)      7m58s
```
* the log of kube-ovn-cni-8lx2p
```shell
$ kubectl logs kube-ovn-cni-8lx2p -n kube-system
ovsdb-server is running with pid 17596
ovs-vswitchd is running with pid 17660
iptables v1.8.7 (legacy)
iptables v1.8.7 (nf_tables):  CHAIN_UPDATE failed (No such file or directory): chain FORWARD
setting sysctl variable "net.ipv4.neigh.default.gc_thresh1" to "1024"
net.ipv4.neigh.default.gc_thresh1 = 1024
setting sysctl variable "net.ipv4.neigh.default.gc_thresh2" to "2048"
net.ipv4.neigh.default.gc_thresh2 = 2048
setting sysctl variable "net.ipv4.neigh.default.gc_thresh3" to "4096"
net.ipv4.neigh.default.gc_thresh3 = 4096
setting sysctl variable "net.netfilter.nf_conntrack_tcp_be_liberal" to "1"
net.netfilter.nf_conntrack_tcp_be_liberal = 1
unknown flag: --ovs-vsctl-concurrency
Usage of ./kube-ovn-daemon:
      --add_dir_header                      If true, adds the file directory to the header of the log messages
      --alsologtostderr                     log to standard error as well as files (no effect when -logtostderr=true) (default false)
      --bind-socket string                  The socket daemon bind to. (default "/run/openvswitch/kube-ovn-daemon.sock")
      --cluster-router string               The router name for cluster router (default "ovn-cluster")
      --cni-conf-dir string                 Path of the CNI config directory. (default "/etc/cni/net.d")
      --cni-conf-file string                Path of the CNI config file. (default "/kube-ovn/01-kube-ovn.conflist")
      --cni-conf-name string                Specify the name of kube ovn conflist name in dir /etc/cni/net.d/, default: 01-kube-ovn.conflist (default "01-kube-ovn.conflist")

```

# Steps to Reproduce the Problem

```shell
wget https://raw.githubusercontent.com/kubeovn/kube-ovn/release-1.12/dist/images/install.sh
bash install.sh
```

# Additional Info

- Kubernetes version:

  ```
  $ kubectl version
  Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.17", 
  GitCommit:"953be8927218ec8067e1af2641e540238ffd7576", GitTreeState:"clean", BuildDate:"2023-02-22T13:34:27Z", 
  GoVersion:"go1.19.6", Compiler:"gc", Platform:"linux/amd64"}
  Server Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.17", 
  GitCommit:"953be8927218ec8067e1af2641e540238ffd7576", GitTreeState:"clean", BuildDate:"2023-02-22T13:27:46Z", 
  GoVersion:"go1.19.6", Compiler:"gc", Platform:"linux/amd64"}
  ```

- kube-ovn version:
  
  ```
  1.12.1
  ```

- operation-system/kernel version:

  ```
  $ awk -F '=' '/PRETTY_NAME/ { print $2 }' /etc/os-release
  "CentOS Linux 7 (Core)"

  $ uname -r
  3.10.0-1160.el7.x86_64
  ```

**Comment by bobz965** [comment]
Thanks for the report, I will get into this

**Comment by qiutingjun** [comment]
Please pull a new image and try again.
It's caused by the image without of commit (https://github.com/kubeovn/kube-ovn/pull/3288) and dose not match the install.sh.

**Comment by oilbeater** [comment]
We have just released a new version 1.12.2, the image contains the new cmd args you can try the new version or delete the `--ovs-vsctl-concurrency` in the scripts with the old version. Thanks again for the report.
